docs/contributors/README.md

```diff
@@ -82,11 +82,41 @@ If you are running into issues or not seeing your updates, a few things you can
 * Delete the `overmind.sock` file and run `script/dev` again
 * Ensure you have run `script/setup`
 * Delete the `node_modules` directory and re-run `script/setup`
-* Run `bundle exec rake docs:build`
-* Run `bundle exec rake docs:preview`
 
 _Note: Overmind is required to run script/dev._
 
+#### Documentation
+
+To run the documentation site locally, one option is to generate the `info_arch.json` file required for the documentation site to run, then updating the path to your generated version in a clone of the [`primer/design` repo](https://github.com/primer/design).
+
+To generate the `info_arch.json` file with your changes to preview, in the `view_components` repo, run:
+
+```bash
+bundle exec rake static:dump_info_arch
+```
+
+You should see a generated `static/info_arch.json` file. Clone the [`primer/design` repo](https://github.com/primer/design) repo and copy/paste the `info_arch.json` file into the root of that repo.
+
+Then, change the path in the [`gatsby-node.esm.js` file](https://github.com/primer/design/blob/main/gatsby-node.esm.js#L199) to point to your new, generated file.
+
+Note: if you're pointing to the file locally (i.e. `./info_arch.json`), you will also need to change the following lines since you don't need to `fetch` this data from an API.
+
+Change:
+
+```bash
+const argsJson = await fetch(url).then(res => res.json())
+const argsContent = JSON.parse(Buffer.from(argsJson.content, 'base64').toString())
+```
+
+to
+
+```bash
+const argsJson = fs.readFileSync(url, 'utf-8')
+const argsContent = JSON.parse(argsJson)
+```
+
+Then, in the **primer/design repo**, run `yarn run start` to spin up the Gatsby docs to preview locally with your changes!
```

Comment:
```suggestion
Then, in the **primer/design repo**, run `npm start` to spin up the Gatsby docs to preview locally with your changes!
```

Reply:
Oh, really? I just ran these steps as I was writing and this worked for me, it's also what is in the [`primer/design` README](https://github.com/primer/design?tab=readme-ov-file#local-development) 🤔

Reply:
I think both will work, but primer/design uses Yarn.

Reply:
Ah interesting! Preference on if I should use `yarn` or `npm` here? I lean towards `yarn` because that's what their own docs say (and I haven't tested w/`npm`)

Reply:
I would prefer us to use npm. But this isn’t a blocker.

Reply:
Normally I would agree, but the project is built with yarn and we mention yarn in our docs. I would like to avoid confusion and use the same tool everywhere. We should absolutely convert the project over to use npm tho 👍